pack_zetasql.sh

```diff
@@ -7,75 +7,136 @@ set -eE
+
+    if [[ "$OSTYPE" == "linux-gnu"* ]]
+    then
+        INSTALL_BIN="install"
+    else
+        INSTALL_BIN="ginstall"
+    fi
+    ${INSTALL_BIN} -D "$file" "$ROOT/tmp-lib/$libname"
 }
 
 install_gen_include_file() {
     local file
     file=$1
     local outfile
     outfile=$(echo "$file" | sed -e 's/^.*proto\///')
-    install -D "$file" "$PREFIX/include/$outfile"
+    
+    if [[ "$OSTYPE" == "linux-gnu"* ]]
+    then
+        INSTALL_BIN="install"
+    else
+        INSTALL_BIN="ginstall"
+    fi
+    ${INSTALL_BIN} -D "$file" "$PREFIX/include/$outfile"
 }
 
 install_external_lib() {
     local file
     file=$1
     local libname
     libname=$(basename "$file")
-    install -D "$file" "$PREFIX/lib/$libname"
+    if [[ "$OSTYPE" == "linux-gnu"* ]]
+    then
+        INSTALL_BIN="install"
+    else
+        INSTALL_BIN="ginstall"
+    fi
+    ${INSTALL_BIN} -D "$file" "$PREFIX/lib/$libname"
 }
 
 export -f install_gen_include_file
 export -f install_lib
 export -f install_external_lib
 
 
+if [[ "$OSTYPE" == "linux-gnu"* ]]
+then
+    INSTALL_BIN="install"
+else
+    INSTALL_BIN="ginstall"
+fi
+
 pushd bazel-bin/
-# exlucde test so
-find zetasql -maxdepth 4 -type f -iname '*.so' -exec bash -c 'install_lib $0' {} \;
+if [[ "$OSTYPE" == "linux-gnu"* ]]
+then
+    # exlucde test so
+    find zetasql -maxdepth 4 -type f -iname '*.so' -exec bash -c 'install_lib $0' {} \;
+fi
 find zetasql -type f -iname '*.a' -exec bash -c 'install_lib $0' {} \;
 
 # external lib headers
 pushd "$(realpath .)/../../../../../external/com_googlesource_code_re2"
-find re2 -iname "*.h" -exec install -D {} "$PREFIX"/include/{} \;
+find re2 -iname "*.h" -exec ${INSTALL_BIN} -D {} "$PREFIX"/include/{} \;
+popd
+
+pushd "$(realpath .)/../../../../../external/com_googleapis_googleapis"
+find google -iname "*.h" -exec ${INSTALL_BIN} -D {} "$PREFIX"/include/{} \;
+popd
+
+pushd "$(realpath .)/../../../../../external/com_google_file_based_test_driver"
+find file_based_test_driver -iname "*.h" -exec ${INSTALL_BIN} -D {} "$PREFIX"/include/{} \;
 popd
 
 # external lib
 pushd external
-find icu -type f -iregex ".*/.*\.\(so\|a\)\$" -exec bash -c 'install_external_lib $0' {} \;
-find com_googlesource_code_re2 -type f -iregex ".*/.*\.\(so\|a\)\$" -exec bash -c 'install_external_lib $0' {} \;
-find com_googleapis_googleapis -type f -iname '*.so' -exec bash -c 'install_external_lib $0' {} \;
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]
+then
+    find icu -type f -iregex ".*/.*\.\(so\|a\)\$" -exec bash -c 'install_external_lib $0' {} \;
+    find com_googlesource_code_re2 -type f -iregex ".*/.*\.\(so\|a\)\$" -exec bash -c 'install_external_lib $0' {} \;
+    find com_googleapis_googleapis -type f -iregex ".*/.*\.\(so\|a\)\$" -exec bash -c 'install_external_lib $0' {} \;
+    find com_google_file_based_test_driver -type f -iregex ".*/.*\.\(so\|a\)\$" -exec bash -c 'install_external_lib $0' {} \;
+else
+    find icu -type f -iname '*.a' -exec bash -c 'install_external_lib $0' {} \;
```

Comment:
the ZetaSQL archive do not include abseil-cpp on linux, instead abseil-cpp is bundled in hybridsql-asserts.
Will this ok for Mac ?